ui/app/src/modules/Preview/QuickCreate.tsx

```diff
@@ -128,16 +130,16 @@ export default function QuickCreate() {
           />
         </MenuItem>
         <Divider />
-        <MenuItem className={classes.menuSectionTitle}>
+        <Typography variant="h4" className={classes.menuSectionTitle}>
```

Comment:
change component prop too

Reply:
I actually removed the variant prop and replaced it with the component prop since I do not need the styles from the h4